packages/markdown-docx/src/ToOOXMLVisitor/index.js

```diff
@@ -59,6 +63,46 @@ class ToOOXMLVisitor {
         return node.$class;
     }
 
+    /**
+     * Generates the OOXML for text and code ciceromark nodes.
+     *
+     * @param {string} value           Text value of the node
+     * @param {Array}  nodeProperties  Properties of the node
+     * @param {string} defaultProperty Default property of the node
+     * @returns {striing} Generated OOXML
+     */
+    generateTextOrCodeOOXML(value, nodeProperties, defaultProperty = '') {
```

Comment:
I don't think so defaultProperty should be an optional parameter. Do you have any case where `''` is a valid argument? Moreover, its name is also not descriptive. I think it should be renamed to `stylisticProperties`.

Reply:
@algomaster99 
In `code` only there is a requirement of `defaultProperty`.For rest of the text nodes, its empty.
I will change some arguments  to make it more understandable.